redhat-director-scripts/docker/trilio-datamover/nova-sudoers

```diff
@@ -1 +1 @@
-nova        ALL=(ALL)       NOPASSWD: ALL
+nova ALL = (root) NOPASSWD: /home/tvault/.virtenv/bin/privsep-helper *
```

Comment:
In RHOSP this path is same?

Reply:
It should be no different than normal redhat/centos setup.

Reply:
If you are sure then ok.